src/lib/components/OpenWebSearchResults.svelte

```diff
@@ -47,7 +53,7 @@
 				{#if sources}
 					Completed
 				{:else}
-					{lastMessage.message}
+					{"message" in lastMessage ? lastMessage.message : "An error occurred"}
```

Comment:
```suggestion
					{lastMessage?.message ?? "An error occurred"}
```

Reply:
We have to do a check with `in` because the message type might not include `message`. Typescript throws an error if we don't check